docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3726 @@
+
+If a non-privileged user is listening to music alone in the channel, they can use commands without restrictions,
+even if DJ role or voteskip settings are active.
+
+.. _sound-quality-issues:
+
+--------------------
+Sound Quality Issues
+--------------------
+
+Laggy audio is most likely caused by:
+
+* A problem with the connection between the host machine of your bot to the Discord server.
+* Issues with Discord.
+
+You can try the following to resolve poor sound quality:
+
+* Don't host on home internet, especially over a WiFi connection, try hosting your bot elsewhere.
+* Try the web browser instead of the desktop client for listening.
+* Simply wait, as audio quality may improve in due course.
+* Restart your bot.
+* Check to make sure it's not just a bad quality song (try a different song).
+* Try to listen on a different Discord server or server region.
+* If not everyone is experiencing the issue, it's a discord client issue.
+
+.. _no-sound:
+
+^^^^^^^^
+No Sound
+^^^^^^^^
+
+If the bot's speaking light is active, but there is no sound, troubleshoot the following:
+
+1. Is the bot's user volume turned up? (right click on the bot in discord, see the slider).
+2. Is the bot muted or deafened? Are you deafened? Are you deaf?
+3. Check Discord audio device settings and volume (cog icon next to your username in the bottom left, click "Voice and Video").
+4. Try dragging and dropping the bot back to the voice channel.
+5. Check system audio device settings and volume.
+6. Ask another member to come into the voice channel to confirm that it's not just you.
+
+If the track progress is stuck on 00:00 when you run ``[p]now``:
+
+1. Try to run ``[p]disconnect`` and replay the song.
+2. Try to reload the audio cog with ``[p]audioset restart``.
+3. Make sure the firewall on the host is configured properly.
+
+-------------------------
+Lavalink - Advanced Usage
+-------------------------
+
+.. _multibots:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Setting up Multiple Red Instances with Audio
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are wanting to use multiple bots with Audio on the same machine, you'll need to make a few
+necessary modifications.
+
+Firstly, stop all Red bots. For each bot using Audio:
+
+1. Start the bot.
+2. Run the command ``[p]llset external``.
+3. Stop the bot.
+
+Next, open a command prompt/terminal window. Navigate to ``<datapath>/cogs/Audio`` for any of your bot
+instances - it doesn't matter which bot as all your bots will now use this single instance of Lavalink.
+You can find your ``<datapath>`` with the ``[p]datapath`` command.
+
+Now you need to smartly determine your RAM needs. If your bot has 1GB RAM available, Lavalink should be
+restricted to perhaps 384MB -> 768MB, depending on the cogs you have installed. If your bot has 2GB of
+RAM available, a good amount may be 512MB -> 1GB. 
+
+Run the following command, where ``Xms`` specifies the initial memory allocation pool and ``Xmx`` is the
```

Comment:
There is no reason to mention Xms it was intentioned removed from the cc because it has no real benefit being there and only server to make the flags more complex